src/util/RouterUtil.js

```diff
@@ -106,6 +107,8 @@ fn.routeAfterAuthStatusChangeError = function (router, err) {
 
   // Token has expired - no longer valid. Navigate back to primary auth.
   if (err.errorCode === ErrorCodes.INVALID_TOKEN_EXCEPTION) {
+    // If token is invalid then it must be deleted from cookie
+    CookieUtil.removeOktaStateToken();
```

Comment:
This looks like it [happens by default](https://github.com/okta/okta-auth-js/blob/236a4a4e54e634a36e004be93a4cda2580198dd2/test/spec/http.js#L263-L276) in AuthJS when we encounter this error code, is that not happening?

cc: @aarongranick-okta